core-plugins/docs/SSHAction-action.md

```diff
@@ -21,15 +21,17 @@ Properties
 
 **user:** The username used to connect to host.
 
-**privateKeyFile:** The file path to Private key.
+**privateKey:** The private key to be used to perform the secure shell action. Users can also specify a macro that will
```

Comment:
it might be better to say `Users can store the private key in CDAP's secure store and use macro to read the private key from the secure store using ...` @awholegunch